addons/event_sale/report/event_event_templates.xml

```diff
@@ -12,9 +12,9 @@
                 <div class="o_event_full_page_ticket_font_faded o_event_full_page_ticket_small_caps font-weight-bold">Order Date</div>
                 <div class="o_event_full_page_ticket_small" t-esc="attendee.sale_order_id.date_order.date()"/>
             </div>
-            <div t-if="attendee and attendee.sale_order_line_id.price_total">
+            <div t-if="attendee and attendee.sale_order_line_id.event_ticket_id.price">
```

Comment:
Just discussed this with Luc, it would be better to display the "price_unit" field of the sale order line instead of the price of the ticket.

That way it's lined up with the price per unit of the sale order, which is more accurate, as it can be changed.

Reply:
Hello @awa-odoo 

Changes done.

Thanks